microchip: Espi reset and oobrst fixs

VenkatKotakonda opened the pull request:
These patches fixes
1. espi bus interrupt miss after reset
2. OOB_RST_WARN interrupt miss

franciscomunoz commented on drivers/espi/espi_mchp_xec.c:
```diff
@@ -574,6 +574,14 @@ static void espi_init_flash(struct device *dev)
+static void espi_bus_init(struct device *dev)
+{
+	const struct espi_xec_config *config = dev->config->config_info;
+	/* Enable interrupts for each logical channel enable assertion */
```
Reword the grammar and introduce a space between 579/580

VenkatKotakonda committed:
driver: espi: Re-enable bus interrupts after espi reset.

Espi bus interrupts need to re-enable after espi reset to get
all espi bus and vw signal interrupts.

Signed-off-by: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>

VenkatKotakonda committed:
driver: espi: Enable OOB_RST_WARN virtual wire interrupt.

Epsi slave need to respond with an OOB_RST_ACK for OOB_RST_WARN sent by
master during host reset. Enable interrupt for OOB_RST_WARN to receive
signal changes and respond with ACK.

Signed-off-by: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
